taxcalc/calcfunctions.py

```diff
@@ -590,86 +860,131 @@ def ItemDed(e17500_capped, e18400_capped, e18500_capped, e19200_capped,
-
-        ID_Medical_frt_add4aged : Addon for medical expenses deduction for
-                                  elderly; addon as a decimal fraction of AGI
-
-        ID_Casualty_frt : Deduction for casualty loss;
-                          floor as a decimal fraction of AGI
-
-        ID_Miscellaneous_frt : Deduction for miscellaneous expenses;
-                               floor as a decimal fraction of AGI
-
-        ID_Charity_crt_all : Deduction for all charitable contributions;
-                             ceiling as a decimal fraction of AGI
-
-        ID_Charity_crt_noncash : Deduction for noncash charitable
-                                 contributions; ceiling as a decimal
-                                 fraction of AGI
-
-        ID_Charity_frt : Disregard for charitable contributions;
-                         floor as a decimal fraction of AGI
-
-        ID_Medical_c : Ceiling on medical expense deduction
-
-        ID_StateLocalTax_c : Ceiling on state and local tax deduction
-
-        ID_RealEstate_c : Ceiling on real estate tax deduction
-
-        ID_AllTaxes_c: Ceiling combined state and local income/sales and
-                       real estate tax deductions
-
-        ID_InterestPaid_c : Ceiling on interest paid deduction
-
-        ID_Charity_c : Ceiling on charity expense deduction
-
-        ID_Charity_f: Floor on charity expense deduction
-
-        ID_Casualty_c : Ceiling on casuality expense deduction
-
-        ID_Miscellaneous_c : Ceiling on miscellaneous expense deduction
-
-        ID_StateLocalTax_crt : Deduction for state and local taxes;
-                               ceiling as a decimal fraction of AGI
-
-        ID_RealEstate_crt : Deduction for real estate taxes;
-                            ceiling as a decimal fraction of AGI
-
-    Taxpayer Characteristics:
-        e17500_capped : Medical expenses, capped by ItemDedCap
-
-        e18400_capped : State and local taxes, capped by ItemDedCap
-
-        e18500_capped : Real-estate taxes, capped by ItemDedCap
-
-        e19200_capped : Interest paid, capped by ItemDedCap
-
-        e19800_capped : Charity cash contributions, capped by ItemDedCap
-
-        e20100_capped : Charity noncash contributions, capped by ItemDedCap
-
-        e20400_capped : Total miscellaneous expenses, capped by ItemDedCap
-
-        g20500_capped : Gross casualty or theft loss (before disregard),
-                        capped by ItemDedCap
+    Parameters
+    ----------
+    e17500_capped: float
+        Schedule A: medical expenses, capped as a decimal fraction of AGI
+    e18400_capped: float
+        Schedule A: state and local income taxes deductlbe, capped as a decimal fraction of AGI
+    e18500_capped: float
+        Schedule A: state and local real estate taxes deductible, capped as a decimal fraction of AGI
+    e19200_capped: float
+        Schedule A: interest deduction deductible, capped as decimal fraction of AGI
+    e19800_capped: float
```

Comment:
It would be useful to note that these variables are capped by the `ItemDedCap` parameter.